### DISCUSSION TITLE
There must be at least 1 Vif as upstream

### DISCUSSION THREAD

**Discussion by Troubadix** [fabianishere/udm-iptv#114] [General]
Hi,

using UDM SE with Firmware 2.4.8

Starting the container fails with error "There must be at leat 1 Vif as upstream".

I used "custom" during installtaion and after that typed in the values from this posting: https://github.com/fabianishere/udm-iptv/discussions/8#discussioncomment-1712342

`root@UDM-SE:~# udm-iptv-diag
Please share the following output with the developers:
=== Configuration ===
WAN Interface: eth8
WAN VLAN: 0 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 232.0.0.0/16 87.141.0.0/16
LAN Interfaces: br4
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: false
=== IP Link and Route ===
=== Service Logs ===
Apr 20 20:33:14 UDM-SE systemd[1]: udm-iptv.service: Consumed 80ms CPU time.
Apr 20 20:33:14 UDM-SE systemd[1]: Started IPTV support for the UniFi Dream Machine.
Apr 20 20:33:14 UDM-SE udm-iptv[21385]: NATing IPTV network ranges (if necessary)
Apr 20 20:33:14 UDM-SE udm-iptv[21385]: Monitoring interface eth8 for changes
Apr 20 20:33:14 UDM-SE udm-iptv[21385]: Setting up igmpproxy
Apr 20 20:33:14 UDM-SE udm-iptv[21385]: Starting igmpproxy
Apr 20 20:33:14 UDM-SE igmpproxy[21385]: There must be at least 1 Vif as upstream.
Apr 20 20:33:14 UDM-SE systemd[1]: udm-iptv.service: Main process exited, code=exited, status=255/EXCEPTION
Apr 20 20:33:14 UDM-SE systemd[1]: udm-iptv.service: Failed with result 'exit-code'.
Apr 20 20:33:14 UDM-SE systemd[1]: udm-iptv.service: Consumed 85ms CPU time.
root@UDM-SE:~#`

Any ideas whats wrong?

Thanks.

**Comment by fabianishere** [comment]
You must have an IP address on the WAN interface for igmpproxy to start.

Try to set `IPTV_WAN_DHCP` to false and `IPTV_WAN_STATIC_IP` to some IP address.

**Reply by samcomi** [reply]
How I can set IPTV_WAN_DHCP to false?
Thanks!

**Comment by Troubadix** [comment]
Where can I find the configuration file for this setting?
EDIT: found in /etc/udm-iptv.conf

What IP-Adresse should I use? From which of my VLANs?

**Reply by fabianishere** [reply]
Looking at the other MagentaTv configurations, you probably need to select a different WAN interface that does have an IP address. Do you by any chance have a `ppp0` interface?

**Reply by Troubadix** [reply]
Yes, there is a ppp0!

I was now able to start the udm-iptv but when I try your troubleshooting-tipps I get
`root@UDM-SE:~# ip -4 addr show dev iptv
Device "iptv" does not exist.
root@UDM-SE:~# ip route list
62.155.242.153 dev ppp0 proto kernel scope link src 46.82.255.180
192.168.1.0/24 dev br0 proto kernel scope link src 192.168.1.1
192.168.2.0/24 dev br2 proto kernel scope link src 192.168.2.1
192.168.3.0/24 dev br3 proto kernel scope link src 192.168.3.1
192.168.4.0/24 dev br4 proto kernel scope link src 192.168.4.1
`
But the diag works
`root@UDM-SE:~# udm-iptv-diag
Please share the following output with the developers:
=== Configuration ===
WAN Interface: ppp0
WAN VLAN: 0 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 232.0.0.0/16 87.141.0.0/16
LAN Interfaces: br4
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: false
=== IP Link and Route ===
35: ppp0: <POINTOPOINT,MULTICAST,NOARP,UP,LOWER_UP> mtu 1492 qdisc pfifo_fast state UNKNOWN group default qlen 3
    inet 46.82.255.180 peer 62.155.242.153/32 scope global ppp0
       valid_lft forever preferred_lft forever
62.155.242.153 proto kernel scope link src 46.82.255.180
=== Service Logs ===
Apr 21 16:49:32 UDM-SE systemd[1]: udm-iptv.service: Consumed 90ms CPU time.
Apr 21 16:49:37 UDM-SE systemd[1]: udm-iptv.service: Service RestartSec=5s expired, scheduling restart.
Apr 21 16:49:37 UDM-SE systemd[1]: udm-iptv.service: Scheduled restart job, restart counter is at 13997.
Apr 21 16:49:37 UDM-SE systemd[1]: Stopped IPTV support for the UniFi Dream Machine.
Apr 21 16:49:37 UDM-SE systemd[1]: udm-iptv.service: Consumed 90ms CPU time.
Apr 21 16:49:37 UDM-SE systemd[1]: Started IPTV support for the UniFi Dream Machine.
Apr 21 16:49:37 UDM-SE udm-iptv[26848]: NATing IPTV network ranges (if necessary)
Apr 21 16:49:37 UDM-SE udm-iptv[26848]: Monitoring interface ppp0 for changes
Apr 21 16:49:37 UDM-SE udm-iptv[26848]: Setting up igmpproxy
Apr 21 16:49:37 UDM-SE udm-iptv[26848]: Starting igmpproxy
root@UDM-SE:~#
`
